content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -93,6 +109,20 @@ the `startingCSV` field of the subscription is not possible when using skiprange
 The above warning means that skiprange by itself is useful for teams who are not interested in
 supporting directly installing versions within a given range.
 
+#### Example for SkipRange
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+The `skipRange` parameter is used to ignore the multiple versions of an operator during the upgrade cycle. In the above example the operator version will be avoided from `4.1.0` to `4.1.1`.
```

Comment:
nit: the there's already an explanation in the section above -- no need to reiterate.

Reply:
Yes, I removed this particular line.